server/media/media.go

```diff
@@ -97,7 +97,7 @@ func matchCORSMethod(allowMethods []string, method string) bool {
 	return false
 }
 
-// CORSHandler is the default CORS processor for use by media handlers.
+// CORSHandler is the default preflight OPTIONS processor for use by media handlers.
```

Comment:
It's not just a preflight processor, it adds CORS headers to regular requests too.